### PR TITLE
Updates to latest user subscription url

### PR DESCRIPTION
**Pull request by rehan99000**

### CODE REVIEW

File: ecommerce/subscriptions/utils.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def get_lms_user_subscription_api_url(subscription_id=None):
     """
     Get LMS url to post user subscription data.
     """
-    return get_lms_url('/api/subscriptions/v1/{}/'.format(subscription_id))
+    return get_lms_url('/api/subscriptions/v1/user_subscriptions/{}/'.format(subscription_id))
```
Comment (ziafazal) [inline ecommerce/subscriptions/utils.py:116]: @rehan99000 
I think this url should be 

`/api/v1/user_subscriptions/{}/'.format(subscription_id))`

Comment (rehan99000) [reply ecommerce/subscriptions/utils.py:116]: I had done this previously. but to fetch LMS resources, ecommerce uses `EdxRestAPIClient` which uses dot syntax to construct URLs. 
https://github.com/edly-io/ecommerce/blob/2af6fde7bafe4cc0e656dc0fe0778694391042ea/ecommerce/subscriptions/utils.py#L54

So, we must use some kind of string in front of `site_configuration.subscriptions_api_client` to make an API call

**Review by ziafazal — APPROVED** [review]
👍

**Review by zubair-arbi — APPROVED** [review]
LGTM 👍

File: ecommerce/subscriptions/utils.py (hunk 1 above)
Comment (zubair-arbi) [reply ecommerce/subscriptions/utils.py:116]: As discussed this URL pattern is being followed by existing API's https://github.com/edly-io/ecommerce/blob/2af6fde7bafe4cc0e656dc0fe0778694391042ea/ecommerce/core/models.py#L489

image (18)

**Pull request merged**